### ISSUE TITLE
Possibility of retraining the model

### ISSUE THREAD

**Issue by RefenRefen** [Varat7v2/PEDApp#3]
Thank you for your great work. Could I train the model with my custom data set?

**Comment by Varat7v2** [comment]
@RefenRefen Yes, you can do that. You just need to annotate your datasets and train YOLO models, whose training and deployment guidelines, you can read on their official github pages.
